### PR TITLE
Please do not Merge practice-nhsnumber-scenario

### PR DESCRIPTION
**Pull request by jbudda**
- Added nhs number scenario in performance tests suite as a trial 
- tests passed locally
- done against branch 'frontend-dean-test'

### CODE REVIEW

**Review by mikeSmith08 — DISMISSED** [review]

**Review by AmarkumarJagtap — DISMISSED** [review]

**Comment by jbudda** [comment]
This is for practice not intended to merge to master.

File: src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala [hunk 1]
```diff
@@ -84,6 +84,27 @@ class ClaimSimulation extends Simulation {
     })
   )
 
+  val randomNhsNos = Iterator.continually(
+    // Random number will be accessible in session under variable "OrderRef"
```
Comment (adeel718) [inline src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala]: Comment mentions OrderRef, also won't be in session this function is just creating the nhsno's

File: src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala [hunk 2]
```diff
@@ -84,6 +84,27 @@ class ClaimSimulation extends Simulation {
...
+    Map("nhsno" -> {
+      randomAlphaNumericString(5)
+    })
+  )
+
+  def randomAlphaNumericString(length: Int): String = {
+    //generates random alphanumeric characters
```
Comment (adeel718) [inline src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala]: Space after /

File: src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala [hunk 3]
```diff
@@ -84,6 +84,27 @@ class ClaimSimulation extends Simulation {
     })
   )
 
+  val randomNhsNos = Iterator.continually(
+    // Random number will be accessible in session under variable "OrderRef"
+    Map("nhsno" -> {
+      randomAlphaNumericString(5)
```
Comment (adeel718) [inline src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala:90]: Should this not be 10 instead of 5, as nhs number is 10 length

Comment (jbudda) [reply src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala:90]: @adeel718 the validation which dean developed on test/nhs-number page allows only 5 characters, more than 5 characters is invalid

File: src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala (hunk 1 above)
Comment (jbudda) [reply src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala]: changed accordingly

File: src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala (hunk 2 above)
Comment (jbudda) [reply src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala]: changed accordingly

**Review by adeel718 — APPROVED** [review]

**Comment by jbudda** [comment]
Closing this pull request as this is only for practice

**Pull request closed without merging**